docs/content/PageLayout.mdx

```diff
@@ -168,6 +168,36 @@ See [storybook](https://primer.style/react/storybook?path=/story/layout-pagelayo
 </Box>
 ```
 
+### With a stickyTop sticky pane
+
+```jsx live
+<Box sx={{height: 320, overflowY: 'auto', border: '1px solid', borderColor: 'border.default'}}>
+  <Box
+    height="64px"
+    width="100%"
+    borderBottom="1px solid"
+    borderColor="border.default"
+    sx={{position: 'sticky', top: '0', backgroundColor: 'canvas.subtle'}}
+    display="flex"
+    alignItems="center"
+    justifyContent="center"
+  >
+    <Heading>Sticky top content</Heading>
+  </Box>
```

Comment:
In Primer React, there are two ways to style the Box component:
* system props (styling props passed directly to the component)
```jsx
<Box width="100%" ...>
```
* `sx` prop
```jsx
<Box sx={{width: '100%'}} ...>
```

In general, we try not to mix system props and the sx prop in one components. We're also slowly moving away from system props, so we prefer using the `sx` prop for all styling. There's more context about the `sx` prop and system props in this ADR if you're curious: https://github.com/primer/react/blob/main/contributor-docs/adrs/adr-003-prop-norms.md
```suggestion
  <Box
    sx={{
      position: 'sticky',
      top: 0,
      height: 64,
      display: 'grid',
      placeItems: 'center',
      backgroundColor: 'canvas.subtle',
      borderBottom: '1px solid',
      borderColor: 'border.default'
    }}
  >
    Custom sticky header
  </Box>
```

Reply:
Thanks for the explanation! And ADR cleared up some of my questions. Thank you!